realm/realm-library/src/main/java/io/realm/RealmObjectSchema.java

```diff
@@ -38,7 +38,7 @@
  */
 public class RealmObjectSchema {
 
-    private static final Map<Class<?>, FieldMetaData> SUPPORTED_SIMPLE_FIELDS;
+    static final Map<Class<?>, FieldMetaData> SUPPORTED_SIMPLE_FIELDS;
```

Comment:
Consider a static method to access this.  Just exposing the implementation is a pretty drastic thing to do.